src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/SQLitePersistor.java

```diff
@@ -179,6 +217,345 @@ synchronized int insertRow(String tableName, List<Object> row) throws SQLExcepti
     return lastPrimaryKey;
   }
 
+  @Override
+  public synchronized <T> T read(Class<T> clz)
```

Comment:
PersistorBase is supposed to be DB agnostic. So, tomorrow we can have a levelDB persistor instead of SQLite and even then PersistorBase would take care of file rotation and gc etc.